package.json

```diff
@@ -35,7 +35,7 @@
     "pretest": "npm run lint"
   },
   "optionalDependencies": {
-    "unix-dgram": "2.0.x"
+    "unix-dgram": "^2.0.3"
```

Comment:
I like to keep this as a patch release, to be more careful with minor releases.  Can you change this back?